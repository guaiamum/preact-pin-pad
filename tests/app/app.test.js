import { h } from 'preact';
import App from '../../src/components/app';
import { render } from 'preact-render-to-string';

test('renders app rightfully', () => {
    const stringified = render(
        <App />,
    );
    expect(stringified).toMatchSnapshot();
});
