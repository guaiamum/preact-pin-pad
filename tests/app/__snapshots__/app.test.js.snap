// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders app rightfully 1`] = `"<div class=\\"vsr\\"></div><div class=\\"keypad\\"><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">1</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">2</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">3</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">4</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">5</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">6</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">7</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">8</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">9</button></div><div class=\\"key-w\\"><button type=\\"button\\" class=\\"key\\">0</button></div></div>"`;
